module TestFailThrow {
    err X;

    private fun Test():(){
        throw .X;
    };
}